src/main/scala/dotty/dokka/tasty/comments/MarkdownConverter.scala

```diff
@@ -32,10 +32,11 @@ class MarkdownConverter(val r: Reflection)(owner: r.Symbol) {
 
   def emitConvertedNode(n: mdu.Node)(using Emitter[dkkd.DocTag]): Unit = n match {
     case n: mda.Paragraph =>
-      if n.getParent.isInstanceOf[mdu.Document]
-      && !Option(n.getPrevious).exists(_.isInstanceOf[mda.Heading])
-      then
-        emit(dkkd.Br.INSTANCE)
+      // if n.getParent.isInstanceOf[mdu.Document]
+      // && !Option(n.getPrevious).exists(_.isInstanceOf[mda.Heading])
+      // then
+      //   emit(dkkd.Br.INSTANCE)
+      // I've commented it because it inserts unnecessary BRs which are breaking layout
```

Comment:
@abgruszecki Can you take a look?

Reply:
This looks OK, but I'd remove those lines instead. They were added in the first place to separate text paragraphs with an empty line, and they don't seem necessary now. Analogous code in wiki.Converter should be cleaned up as well: https://github.com/lampepfl/scala3doc/blob/4963d716c4f4f5484a8806545ecff9657014ba98/src/main/scala/dotty/dokka/tasty/comments/wiki/Converter.scala#L40-L43 (note that the `topLevel` parameter will, I believe, become useless if this code is removed).

This is a good page to see the current look: https://scala3doc.s3.eu-central-1.amazonaws.com/pr-167/self/scala3doc/tests/-a/index.html . Compare with: https://scala3doc.s3.eu-central-1.amazonaws.com/pr-167/self/scala3doc/tests/-b/index.html , which is rendered from Wiki syntax.